definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -18010,6 +18010,34 @@
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "isNocFixAppliedFlag",
+    "UserRole": "caseworker-divorce-systemupdate",
+    "CRUD": "CRUD"
+  },
+  {
```

Comment:
Does any other role need to see this field other than caseworker-divorce-systemupdate?

Reply:
TBF, you're right I can chop the other ones